packages/service-library/tests/fastapi/test_openapi.py

```diff
@@ -8,8 +8,12 @@
 import starlette.routing
 from fastapi.applications import FastAPI
 from fastapi.routing import APIRouter
-from openapi_spec_validator.exceptions import OpenAPISpecValidatorError
-from openapi_spec_validator.shortcuts import get_validator_cls
+from openapi_spec_validator.exceptions import (
+    OpenAPISpecValidatorError,  # pylint: disable=no-name-in-module
+)
+from openapi_spec_validator.shortcuts import (
+    get_validator_cls,  # pylint: disable=no-name-in-module
```

Comment:
minor: for test files, you can do a global pylint disable (as shown at top of this file)